### PR TITLE
Update developer docs to match the current implementation

### PR DESCRIPTION
**Pull request by bob-wilson**
- Fix code samples in move-overview.md so that they will compile
- Update commands and sample output in my-first-transaction.md for
shorter addresses and multicurrency accounts. Also update the output
shown in the "Sample Outputs of Additional Query Commands" section.
- Update run-move-locally.md for multicurrency accounts.

## Motivation

Current docs do not match the implementation

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/website/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

I ran the code samples through the Move compiler and made sure that they compile, and I walked through most of the steps in the documents to check that the commands and sample output match the current status.

### CODE REVIEW

**Comment by bob-wilson** [comment]
Note that there will be more changes coming soon in the Move code....

I'm also wondering if it would be better to remove some or all of the sample output in the "Sample Outputs of Additional Query Commands" section. It's very detailed and likely to be constantly getting out-of-date.

**Comment by JoelMarcey** [comment]
> I'm also wondering if it would be better to remove some or all of the sample output in the "Sample Outputs of Additional Query Commands" section. It's very detailed and likely to be constantly getting out-of-date.

Could we have snippets of the output that would generally be relevant and unchanging?

File: docs/my-first-transaction.md [hunk 1]
```diff
@@ -143,7 +146,7 @@ Sample output on success:
 
 ```plaintext
 >> Creating/retrieving next account from wallet
-Created/retrieved account #0 address 3ed8e5fafae4147b2a105a0be2f81972883441cfaaadf93fc0868e7a0253c4a8
+Created/retrieved account #0 address cc2219df031a68115fad9aee98e051e9
```
Comment (JoelMarcey) [inline docs/my-first-transaction.md:149]: Could also think about just having a placeholder address field here...

```
<SOME_ADDRESS>
```

Unless we think the address format is pretty stable now?

**Comment by bob-wilson** [comment]
> > I'm also wondering if it would be better to remove some or all of the sample output in the "Sample Outputs of Additional Query Commands" section. It's very detailed and likely to be constantly getting out-of-date.
> 
> Could we have snippets of the output that would generally be relevant and unchanging?

I don't know but probably not. Another option would be to show the commands and describe what kind of output they show, without the verbatim sample output.

File: docs/my-first-transaction.md (hunk 1 above)
Comment (bob-wilson) [reply docs/my-first-transaction.md:149]: > Unless we think the address format is pretty stable now?

The address format should be stable from now on.

**Pull request merged**